frontend/src/metabase/public/components/EmbedModal/SelectEmbedTypePane/SelectEmbedTypePane.tsx

```diff
@@ -171,13 +201,13 @@ export function SelectEmbedTypePane({
             >{t`Settings`}</Link>
           </Text>
         )}
-        <a
+        <ExternalLink
           className={cx(CS.link, CS.textBold)}
           style={{ display: "flex", alignItems: "center", gap: 4 }}
           href={embeddingUrl}
         >
-          {t`Compare options`} <Icon name="share" />
-        </a>
+          {t`Compare options`} <Icon name="share" aria-hidden />
```

Comment:
`aria-hidden` are to make it more accessible and and we could use accessible queries in testing.